tasks/qcow2_image.yml

```diff
@@ -94,6 +94,16 @@
       tags:
         - ovirt-template-image
 
+    - name: Wait until the qcow image is unlocked by the oVirt engine
+      ovirt_disk_facts:
+        auth: "{{ ovirt_auth }}"
+        pattern: "id={{ ovirt_disk.id }}"
+      until: (ovirt_disks is defined) and (ovirt_disks[0].status != "locked")
+      retries: 10
+      delay: 3
```

Comment:
Ondro, is it worth to have retries/delay configurable as role paramater?
@machacekondra

Reply:
Actually I think this is bug here:

https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/ovirt/ovirt_disk.py#L388

The condition is wrong. We should wait in case of not lun disk.

Reply:
So, does that mean that the proper solution is to patch the ovirt_disk module, instead of trying to workaround the problem in this role?

Reply:
Yeah, I think it would be much better solution.

Reply:
@machacekondra a colleague hit a similar bug while doing disk operations, so I just want to confirm what is the next step here: will you patch the ovirt_disk module, or am I supposed to present a PR?

Reply:
@machacekondra Probably the same issue was encountered (the difference only in ok status of disk). The process of creating the VM template starting before image upload finishes and it cause "Failed to create Template" in most time (approximately 1 successful template creation per 3 tries). Should this issue be delegated to ovirt_disk module or some patch will be provided?

Reply:
Sorry for late reply, I've sent fix:

https://github.com/ansible/ansible/pull/55426

I will merge this one as well, and when bump on Ansible 2.8 I will remove it. Thanks!